benchexec/tools/aise.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+import benchexec.model
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for AISE.
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("aise", subdir="bin")
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
+        )
+
+    def cmdline(self, executable, options, task, rlimits):
+        return [executable] + options + list(task.input_files_or_identifier)
```

Comment:
Does your tool support tasks with several input files? Does your tool support tasks without any input file at all? If yes, you can keep it, otherwise please use one of the other properties than `task.input_files_or_identifier`.

Reply:
No, it just need single input file. I fixed it.